include/fiction/algorithms/simulation/sidb/is_ground_state.hpp

```diff
@@ -41,7 +42,8 @@ template <typename Lyt>
     const auto min_energy_exact  = minimum_energy(exhaustive_results.charge_distributions);
     const auto min_energy_new_ap = minimum_energy(quicksim_results.charge_distributions);
 
-    return std::abs(min_energy_exact - min_energy_new_ap) / min_energy_exact < physical_constants::POP_STABILITY_ERR;
+    return round_to_n_decimal_places((min_energy_exact - min_energy_new_ap).value(), 6) <
```

Comment:
Is `std::abs`/`units::math::abs` no longer needed?

Reply:
It was an old version before. I used ``std::abs`` before we had ``round_to_n_decimal_places``.